ui-manchette-with-spacetimechart/src/helpers.ts

```diff
@@ -0,0 +1,166 @@
+import type {
+  OperationalPointType,
+  ProjectPathTrainResult,
+  StyledOperationalPointType,
+} from '@osrd-project/ui-manchette/dist/types';
+import type {
+  OperationalPoint,
+  SpaceTimeChartProps,
+} from '@osrd-project/ui-spacetimechart/dist/lib/types';
+
+import { BASE_OP_HEIGHT, MAX_TIME_WINDOW, MAX_ZOOM_X, MIN_ZOOM_X } from './consts';
+import { getHeightWithoutLastWaypoint, msToS } from './utils';
+
+export const calcOperationalPointsToDisplay = (
```

Comment:
Maybe `filterWaypointsToDisplay` would be a better name
```suggestion
export const filterWaypointsToDisplay = (
```

Reply:
Since the refacto with solely the point display push in the result, i think it would be better to do it at this time, because for now they are all returned